### PR TITLE
typos, markup

### PR DESCRIPTION
**Pull request by MarkowEduard**

### CODE REVIEW

File: BugFixes/README.md [hunk 1]
```diff
@@ -15,16 +14,13 @@ exported variables may fail. Enforce this in the compiler.
 Fix 2: Initializing of `GraphicFrames.TBuf` variable
 ----------------------------------------------------
 
-
-When you delete text from a Drawing, the extra letters get moved into a `Texts.Buffer` called `Tbuf`.
+If you delete text from a Drawing, the extra letters get moved into a `Texts.Buffer` called `Tbuf`.
```
Comment (schierlm) [inline BugFixes/README.md:17]: When/If in this case (general condition, "whenever") is not a typo, but a matter of style. Compare https://www.bbc.co.uk/worldservice/learningenglish/grammar/learnit/learnitv261.shtml#:~:text=when%20and%20if%20are%20interchangeable,general%20conditions

File: BugFixes/README.md [hunk 2]
```diff
@@ -15,16 +14,13 @@ exported variables may fail. Enforce this in the compiler.
...
 Only that this buffer is never initialized, resulting in memory corruption if you delete enough text.
 
-
 Fix 3: Avoid freelist corruption after memory allocation failure
 ----------------------------------------------------------------
 
-
-When a memory of a small type from a large block fails, the system will still store
+If allocation of a small amount from a large block fails, the system will still store
```
Comment (schierlm) [inline BugFixes/README.md:23]: Same.

File: BugFixes/README.md [hunk 3]
```diff
@@ -33,7 +29,7 @@ Fix 4: Avoid illegal IO access (-4) when GC encounters NIL pointers
 
 As discussed on the mailing list (`PO: Illegal memory access in GC`), the garbage collector
 will read from IO address -4 when it encounters a NIL pointer and tries to read the metadata
-preceding the pointer. By default, IO address -4 is unused and will return 0, but better
+preceding the pointer. By definition, IO address -4 is unused and will return 0, but better
```
Comment (schierlm) [inline BugFixes/README.md:32]: I will have to reword this paragraph anyway. IO address space is board/hardware dependent and no definition. I was referring to the configuration on the original board and/or compatible ones.

**Review by schierlm — COMMENTED** [review]
See my review below.

The rest looks fine. I can cherry pick the other fixes, or you can update the pull request. What do you prefer?

File: BugFixes/README.md (hunk 3 above)
Comment (MarkowEduard) [reply BugFixes/README.md:32]: Cherry picking :-)
All of those small change proposals are not really necessary. If one or all don't fit, just drop them.

**Comment by MarkowEduard** [comment]
Cherry picking :-)
All of those small change proposals are not really necessary. If one or all don't fit, just drop them.

**Comment by schierlm** [comment]
Applied in 74380023da784e84737668555d726acf888bbba9

**Pull request closed without merging**